Source/DateTimePicker.swift

```diff
@@ -14,6 +14,12 @@ public protocol DateTimePickerDelegate: AnyObject {
 
 @objc public class DateTimePicker: UIView {
     
+    public var doneButtonIsEnabled: Bool = true {
+            didSet {
+                doneButton.isEnabled = doneButtonIsEnabled
+            }
+    }
```

Comment:
A few nits:
- I think it's nicer if we use a method instead of this long awkward variable name. So instead we can have something like: `func enableDoneButton(_ enabled: Bool)`.
- The indenting doesn't quite match with the rest of the file, can you please update it?
- When the done button is disabled, I find that it still looks the same as when it's enabled so it doesn't look obvious and can be mistaken with an issue. How about updating the button's look (i.e make it grey) when it's disabled?